spec/unit/provider/service/systemd_spec.rb

```diff
@@ -372,6 +372,29 @@
     end
   end
 
+  describe "#insync_enabled?" do
+    provider = described_class.new(Puppet::Type.type(:service).new(:name => 'sshd.service', :enable => false))
```

Comment:
You can use a `let` to define this and it will be lazy evaluated.
http://www.betterspecs.org/#let